sections/connected_projects.md

```diff
@@ -0,0 +1,145 @@
+| use_sprints                  | Boolean                |
+| sprint_length                | Integer                |
+| minutes_per_estimation_point | Integer                |
+| created_by                   | Integer, ID of person  |
+| updated_by                   | Integer, ID of person  |
+| created_at                   | Date                   |
+| updated_at                   | Date                   |
+
+### Sample JSON response
+
+```javascript
+[
+   {
+      "id":1,
+      "company_connected_project_id":1,
+      "name":"Website connected project",
+      "color": "#E3E190",
+      "estimation_units": "POINTS",
+      "use_sprints": true,
+      "sprint_length": 7,
+      "minutes_per_estimation_point": 480,
+      "created_by":1,
+      "updated_by":1,
+      "created_at":"2017-01-14T18:46:56Z",
+      "updated_at":"2017-01-14T18:47:58Z"
+   }, ...
+]
+```
+
+## Get connected project
+
+* `GET /connected_projects/{connectedProjectId}` - Returns a specific connected project.
+
+| Response fields              | Description/format     |
+| ---------------------------- | ---------------------- |
+| id                           | Integer                |
+| company_connected_project_id | Integer                |
+| name                         | String                 |
+| color                        | String                 |
+| estimation_units             | String (HOURS, POINTS) |
+| use_sprints                  | Boolean                |
+| sprint_length                | Integer                |
+| minutes_per_estimation_point | Integer                |
+| created_by                   | Integer, ID of person  |
+| updated_by                   | Integer, ID of person  |
+| created_at                   | Date                   |
+| updated_at                   | Date                   |
+
+### Sample JSON response
+
+```javascript
+{
+     "id":1,
+     "company_connected_project_id":1,
+     "name":"Website connected project",
+     "color": "#E3E190",
+     "estimation_units": "POINTS",
+     "use_sprints": true,
+     "sprint_length": 7,
+     "minutes_per_estimation_point": 480,
+     "created_by":1,
+     "updated_by":1,
+     "created_at":"2017-01-14T18:46:56Z",
+     "updated_at":"2017-01-14T18:47:58Z"
+}
+```
+
+## Create connected project
+
+* `POST /connected_projects` - Creates a new connected project. Returns the same object as getting a single project.
+
+| Request fields               | Description/format     |
+| ---------------------------- | ---------------------- |
+| name                         | String                 |
+| color                        | String                 |
+| estimation_units             | String (HOURS, POINTS) |
+| use_sprints                  | Boolean                |
+| sprint_length                | Integer                |
+| projects                     | Array, IDs of projects |
+| minutes_per_estimation_point | Integer                |
```

Comment:
Add defaults to the fields that make sense. Estimation units, use sprints, sprint length and minutes per estimation point